.github/actions/run-eden-test/action.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Run specified Eden test workflow'
+description: 'Setup Eden run specified test from workflow and publish logs'
+
+inputs:
+  file_system:
+    required: true
+    type: string
+  tpm_enabled:
+    required: true
+    type: bool
+  suite:
+    required: true
+    type: string
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup Environment
+      uses: ./.github/actions/setup-environment
+      with:
+        file_system: ${{ inputs.file_system }}
+        tpm_enabled: ${{ inputs.tpm_enabled }}
+    - name: Run tests
+      run: EDEN_TEST_STOP=n ./eden test ./tests/workflow -s ${{ inputs.suite }} -v debug
+      shell: bash
+    - name: Collect logs
+      uses: ./.github/actions/publish-logs
```

Comment:
We should use `if: always()` here.
Actually I cannot find any logs in your [sample](https://github.com/uncleDecart/eden/actions/runs/5197858608), am I missing something?

Reply:
done